test/demo/utils/storageRemote.ts

```diff
@@ -3,37 +3,29 @@ import {Identity} from 'hadouken-js-adapter';
 import {SERVICE_IDENTITY} from '../../../src/client/internal';
 import {StoredNotification} from '../../../src/provider/model/StoredNotification';
 import {Notification} from '../../../src/client';
+import {Database, CollectionMap} from '../../../src/provider/model/database/Database';
 
 import {OFPuppeteerBrowser, BaseWindowContext} from './ofPuppeteer';
 
-type LocalForage = typeof import('localforage');
-
 interface ProviderWindow extends BaseWindowContext {
-    settingsStorage: LocalForage;
-    notificationStorage: LocalForage;
+    database: Database;
 }
 
 const ofBrowser = new OFPuppeteerBrowser<ProviderWindow>();
 
 export async function getAllStoredNotifications(): Promise<StoredNotification[]> {
     return ofBrowser.executeOnWindow(SERVICE_IDENTITY, async function(): Promise<StoredNotification[]> {
-        const allNotes: StoredNotification[] = [];
-        await this.notificationStorage.iterate((val: string) => {
-            allNotes.push(JSON.parse(val));
-        });
+        const allNotes: StoredNotification[] = await this.database.get(CollectionMap.NOTIFICATIONS).getAll();
```

Comment:
I just want to jump in and say how much nicer this syntax is. I really hated the old one.

Reply:
Thanks buddy